client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -307,9 +322,12 @@ const mapStateToProps = ( state ) => {
 		progressLoaded: getUploadProgressLoaded( state, siteId ),
 		installing: isInstallInProgress( state, siteId ),
 		backPath: getBackPath( state ),
-		showEligibility: ! isJetpack && ( hasEligibilityMessages || ! isEligible ),
+		showEligibility:
+			( ( isJetpack && isAtomic ) || ( ! isJetpack && ! isAtomic ) ) &&
```

Comment:
Minor: all Atomic sites are Jetpack sites, so I think this can be simplified to:
```suggestion
			( isAtomic || ! isJetpack ) &&
```

Reply:
Thank you. I was staring at this for too long to be able to simplify it. Done in https://github.com/Automattic/wp-calypso/pull/52776/commits/86feb063a09b0d642a52eefd2b5cc180250e30d5

Reply:
I retested the logic and it works as expected still.